docs/_content/themas/achtergronddocumentatie/versies-en-migraties.md

```diff
@@ -44,25 +48,24 @@ HTTP 200
 }
 ```
 
-1. Huidige situatie: Zaken API v1.0.0
-2. Een Zaak heeft een Status (beide in de Zaken API)
-3. Zaken API v2.0.0 komt uit
+3. De provider heeft er voor gekozen de URI's naar gerelateerde objecten in een koppeltabel in de database vast te leggen.
+4. De provider gaat de Zaken API v2.0.0 ondersteunen en zal op termijn gaan stoppen met de ondersteuning van v1.0.0.
 
 **Wat gaat er gebeuren?**
 
-1. Zaken API v1.0.0 zal nog even moeten bestaan naast v2.0.0
-2. Verzoeken op de Zaken API v1.0.0 geven een waarschuwing mee als header
-   dat er een nieuwe versie is.
-3. Zaak met UUID `6c821f` moet beschikbaar zijn op de Zaken API v1.0.0 en 
-   v2.0.0
-4. Verwijzingen binnen dezelfde API wijzen naar dezelfde versie. Ofwel, een 
+1. De provider zal de Zaken API v1.0.0 nog enige tijd naast v2.0.0 moeten ondersteunen.
+2. Vanaf het beschikbaar komen van de Zaken API v2.0.0 geven verzoeken op de Zaken API v1.0.0 een waarschuwing mee als header
+   waarin staat dat er een nieuwe versie beschikbaar is.
+3. In de koppeltabel is voor het object met de UUID's '8aa57d' vastgelegd welke v1.0.0 en welke v2.0.0 URI van toepassing is. 
+4. De zaak met UUID `6c821f` is beschikbaar via de Zaken API v1.0.0 en via v2.0.0.
+5. Verwijzingen binnen dezelfde API wijzen naar dezelfde versie. Ofwel, een 
    Zaak in Zaken API v1.0.0 mag niet wijzen naar een Status in de Zaken API 
-   v2.0.0.
-5. Voordat Zaken API v1.0.0 wordt stopgezet moeten alle applicaties (die
+   v2.0.0. Door het in punt 4 genoemde is dat ook mogelijk.
+6. Voordat de Zaken API v1.0.0 wordt stopgezet moeten alle applicaties (die
    iets doen met de Zaken API) kunnen omgaan met Zaken API v2.0.0.
-6. Als alle applicaties kunnen omgaan met Zaken API v2.0.0 kunnen alle URL's
-   naar de Zaken API v1.0.0 worden aangepast naar de nieuwe versie
-7. Zaken API v1.0.0 kan worden stopgezet
+7. Als alle applicaties kunnen omgaan met Zaken API v2.0.0 worden in de Zaken API v1.0.0 alleen de GET endpoints nog ondersteund.
+8. In de koppeltabel worden alle URI's naar de Zaken API v1.0.0 verwijderd.
```

Comment:
Wacht. Wat betekent dit? Moeten ze niet omgezet worden?

Reply:
Van omzetten is nu geen sprake, dat realiseer je m.b.v. de koppeltabel. Zodra er dus niemand meer is die de v1.0.0 verwijzingen in de koppeltabel gebruikt kunnen deze verwijderd worden.

Reply:
Nu je de koppeltabel hebt uitgelegd begrijp ik wat je bedoeld!